CompilerRuntime/InWorldz.Phlox/Compiler/DefaultConstants.cs

```diff
@@ -239,6 +239,22 @@ public class DefaultConstants
             {"ATTACH_HUD_BOTTOM_RIGHT", new ConstantSymbol("ATTACH_HUD_BOTTOM_RIGHT", SymbolTable.INT, "38")},
             {"ATTACH_NECK", new ConstantSymbol("ATTACH_NECK", SymbolTable.INT, "39")},                      // identifier name not yet final https://jira.secondlife.com/browse/SCR-217
             {"ATTACH_AVATAR_CENTER", new ConstantSymbol("ATTACH_AVATAR_CENTER", SymbolTable.INT, "40")},    // identifier name not yet final https://jira.secondlife.com/browse/SCR-217
+            // Bento Additions
+            {"ATTACH_LHAND_RING1", new ConstantSymbol("ATTACH_LHAND_RING1", SymbolTable.INT, "41")},
+            {"ATTACH_RHAND_RING1", new ConstantSymbol("ATTACH_RHAND_RING1", SymbolTable.INT, "42")},
```

Comment:
I was sure that I saw RING2 in one of the lists in my Google search of the SL site, but I cannot find it now. Mike has confirmed these constants compile in SL, so that is the only thing that matters here. So this is ready to merge.